auto_nag/scripts/component.py

```diff
@@ -48,6 +48,9 @@ def get_bz_params(self, date):
             # Ignore bugs for which somebody has ever modified the product or the component.
             'n1': 1, 'f1': 'product', 'o1': 'changedafter', 'v1': '1970-01-01',
             'n2': 1, 'f2': 'component', 'o2': 'changedafter', 'v2': '1970-01-01',
+            'f3': 'bug_status',
+            'o3': 'anywordssubstr',
+            'v3': 'UNCONFIRMED,NEW',
```

Comment:
Maybe `bug_status=UNCONFIRMED&bug_status=NEW` could be faster to handle for Bugzilla's DB?

Reply:
I've absolutely no idea about that... but good question :)
@dylanwh, wdyt ?

Reply:
(or, if you want to use the fields, `anyexact` should be better than `anywordssubstr`)

Reply:
@marco-c I agree here